### PR TITLE
Dark mode

### PR DESCRIPTION
**Pull request by RblSb**
Let me know if something bad

### CODE REVIEW

File: themes/default/templates/main.mtt [hunk 1]
```diff
@@ -12,17 +12,47 @@
 
 ::use "header.mtt"::::end::
 <body>
+<script>
```
Comment (Gama11) [inline themes/default/templates/main.mtt:15]: `index.js` is possibly a better place to put this?

**Comment by anissen** [comment]
This PR would be even greater if extended to automatically choose the system light/dark mode preference (see https://css-tricks.com/a-complete-guide-to-dark-mode-on-the-web/#os-level and https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) 😉

**Comment by Gama11** [comment]
@RblSb Thanks for the contribution!
@anissen Implemented your suggestion.

**Comment by RblSb** [comment]
What should be changed to add dark mode toggler to api.haxe.org?

**Pull request merged**